tests/test_db_util.py

```diff
@@ -5,7 +5,7 @@
 def test_alembic_version(db):
     from clld.db.util import set_alembic_version, get_alembic_version
 
-    assert get_alembic_version(db) is None
+    assert get_alembic_version(db) != '1234'
```

Comment:
Looks like `set_alembic_version` bypasses the session (directly using autocomitting engine), not sure if that is intended:
https://github.com/clld/clld/blob/9ec79af8abb0277a25242332b69445b0e49a2bcd/src/clld/db/util.py#L154
http://docs.sqlalchemy.org/en/latest/orm/session_api.html#sqlalchemy.orm.session.Session.execute

Reply:
Since we dropped the support for (un)freezing databases, this functionality is probably obsolete altogether ...